bridge/client/app/_models/trace.ts

```diff
@@ -195,16 +198,19 @@ class Trace {
       if(this.isProblem() && this.isProblemResolvedOrClosed()) {
         this.label = EVENT_LABELS[EventTypes.PROBLEM_RESOLVED];
       } else if(this.isApprovalFinished()) {
-        this.label = EVENT_LABELS[EventTypes.APPROVAL_FINISHED][this.data.approval?.result] || this.type;
+        this.label = EVENT_LABELS[EventTypes.APPROVAL_FINISHED][this.data.approval?.result] || this.type.split(".").slice(-2)[0];
       } else {
-        this.label = EVENT_LABELS[this.type] || this.type;
+        this.label = EVENT_LABELS[this.type] || this.type.split(".").slice(-2)[0];
```

Comment:
[Suggestion] ```this.type.split(".").slice(-2)[0];``` could be extracted to a function so it's a bit easier to understand what it extracts (also in line 201)

Reply:
Thanks. Extracted it to a function and actually changed also the returned value.